src/components/FieldImage.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import Dropzone from 'react-dropzone';
+import { Field } from 'redux-form';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { getTranslator } from '../reducers/translation';
+import styles from './FieldImage.scss';
+
+const fileInstance = new FileReader(); // eslint-disable-line no-undef
+
+const DropzoneImage = ({ i18n, input }) => (
+  <div>
+    <Dropzone
+      accept={'image/*'}
+      maxSize={300000}
+      multiple={false}
+      className={styles.overide}
+      name={input.name}
+      onDrop={(imageData) => {
+        fileInstance.readAsDataURL(imageData[0]);
+        fileInstance.onloadend = () => input.onChange(fileInstance.result);
```

Comment:
Do you mean fileInstance?
Currently if we have multiple <FieldImage> components, they both work independently and both will store URL images in the state